qualification/Jenkinsfile

```diff
@@ -73,20 +73,33 @@ pipeline {
          script {
```

Comment:
Since we're not writing general Groovy code (try/finally) we don't need to wrap these steps in a `script` block any more.